test/unit/utils/model-utils.test.ts

```diff
@@ -0,0 +1,54 @@
+import type { Sequelize } from '@sequelize/core';
+import { isModelStatic, isSameInitialModel, Model } from '@sequelize/core';
+// eslint-disable-next-line import/order
+import { expect } from 'chai';
+
+const sequelize: Sequelize = require('../../support').sequelize;
+
+describe('isModelStatic', () => {
+  it('returns true for model subclasses', () => {
+    const MyModel = sequelize.define('myModel', {});
+
+    expect(isModelStatic(MyModel)).to.be.true;
+  });
+
+  it('returns true for model instances', () => {
```

Comment:
```suggestion
  it('returns false for model instances', () => {
```

Or is the test incorrect?

Reply:
The title was wrong :)